lib/js/Animation.js

```diff
@@ -34,7 +34,7 @@ const ViewStyleExceptBorderPropType = (props, propName, componentName, ...rest)
       'to render a border around this component, wrap it with a View.'
     );
   }
-  return ViewPropTypes.style(props, propName, componentName, ...rest);
+  return (ViewPropTypes || View.propTypes).style(props, propName, componentName, ...rest);
```

Comment:
This will cause warnings on new RN versions
https://github.com/facebook/react-native/blob/e10f7788c4ebf29d96594f9753264bdf6146f0cf/Libraries/ReactNative/verifyPropTypes.js#L41